cmd/albatross/albatross.go

```diff
@@ -66,3 +70,9 @@ func serveDocumentation(r *mux.Router) {
 		r.PathPrefix("/docs/").Handler(http.StripPrefix("/docs/", fs))
 	}
 }
+
+func handleRepositoryRoutes(router *mux.Router) {
+	repoClient := helmcli.NewRepoClient()
+	repoService := repository.NewService(repoClient)
+	router.Handle(fmt.Sprintf("/{%s}", repository.NAME), ContentTypeMiddle(repository.AddHandler(repoService))).Methods(http.MethodPut)
```

Comment:
The albatross chart has a configmap, which populates the repo file based on values provided to helm. This will create a drift, if repos are managed via Helm.

Reply:
We will need to be explicit about this in the chart, as the drift will not be visible in the diff

Reply:
Yes.